src/components/fieldset/fieldset.ts

```diff
@@ -0,0 +1,29 @@
+import { BaseComponent, BaseElement, BaseOptions } from '../base';
+import { FormioComponents } from '../components';
+import { FormioTemplate } from '../../formio.template';
+import { FormArray, FormGroup, FormControl } from '@angular/forms';
+
+export interface FieldSetOptions extends BaseOptions<any> {
+    components: Array<any>,
+    legend?: string
+}
+
+export class FieldSetComponent extends BaseComponent<FieldSetOptions> {
+    getControl(): FormArray | FormGroup | FormControl {
+        if (!this.control) {
+            this.control = new FormGroup({});
+        }
+        return this.control;
+    }
+}
+
+export class FieldSetElement extends BaseElement<FieldSetComponent> {
+    get numComponents() : number { return this.component.settings.components.length; }
```

Comment:
Please see my other comment @ https://github.com/formio/ng2-formio/pull/11. This should probably return 1 now.